frontend/packages/ux-editor/src/components/config/componentSpecificContent/Image/ImageComponent.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import Select from 'react-select';
-import { PropertyLabel, selectStyles } from '../../../../utils/render';
+import { selectStyles } from '../../../../utils/render';
 import type { FormComponentType, IFormImageComponent } from '../../../../types/global';
 import { FieldSet, TextField } from '@digdir/design-system-react';
 import classes from './ImageComponent.module.css';
 import { TextResource } from '../../../TextResource';
 import { useText } from '../../../../hooks';
+import { Label } from '@altinn/schema-editor/components/common/Label';
```

Comment:
Should this component be moved into 'shared' if we want to use it in multiple packages?